src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,37 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "movie_sessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String name;
```

Comment:
The `MovieSession` class currently only has an `id` and `name` field. Typically, a `MovieSession` would also include fields for the movie it is associated with, the cinema hall, and the session time. Consider adding these fields to fully represent a movie session.